### ISSUE TITLE
Full Update show doesn't seem to update to thexem

### ISSUE THREAD

**Issue by yongbi85** [SiCKRAGETV/sickrage-issues#368]
Hello,

Before the change of design, I could see the name of second,third,... seasons of animes by just moving my mouse over the show name, so I could see the different names of season, however I don't find this option anymore.
And althought a show is referenced in thexem.de when I tryed to post process the episodes of second season episode 01, sickrage said I don't have the show in my list.
So is full update interrogating thexem for changes ? In the log i only see imdb and tvdb

In this particular case i tryed to post process the episode 01 of second season of Tokyo Ghoul.
On thexem.de we can see the different names of second season : 

This is thexem linked from sickrage : http://thexem.de/xem/show/1844
We can see the name of second season Tokyo Ghoul Root A

The log below of the show update and after that the failed post process log

```
2015-01-09 20:25:54 INFO     SHOWQUEUE-REFRESH :: No new cache images needed, not retrieving new ones
2015-01-09 20:25:54 INFO     SHOWQUEUE-REFRESH :: Checking & filling cache for show Tokyo Ghoul
2015-01-09 20:25:54 INFO     SHOWQUEUE-REFRESH :: 281249: Writing NFOs for all episodes
2015-01-09 20:25:54 INFO     SHOWQUEUE-REFRESH :: 281249: Writing NFOs for show
2015-01-09 20:25:54 INFO     SHOWQUEUE-REFRESH :: 281249: Loading all episodes with a location from the database
2015-01-09 20:25:48 INFO     SHOWQUEUE-REFRESH :: 281249: Loading all episodes from the show directory /volume1/Videos/Animes/Tokyo Ghoul
2015-01-09 20:25:48 INFO     SHOWQUEUE-REFRESH :: Performing refresh on Tokyo Ghoul
2015-01-09 20:25:42 INFO     SHOWQUEUE-FORCE-UPDATE :: 281249: Loading all episodes from theTVDB..
2015-01-09 20:25:41 INFO     SHOWQUEUE-FORCE-UPDATE :: Loading all episodes from the DB
2015-01-09 20:25:34 INFO     SHOWQUEUE-FORCE-UPDATE :: 281249: Loading show info from IMDb
2015-01-09 20:25:33 INFO     SHOWQUEUE-FORCE-UPDATE :: 281249: Loading show info from theTVDB
2015-01-09 20:25:33 INFO     SHOWQUEUE-FORCE-UPDATE :: Beginning update of Tokyo Ghoul 


2015-01-09 20:33:47 WARNING  POSTPROCESSER :: Problem(s) during processing
2015-01-09 20:33:47 DEBUG    POSTPROCESSER :: Using ALL regexes
2015-01-09 20:33:47 DEBUG    POSTPROCESSER :: Processing folder @eaDir
2015-01-09 20:33:47 WARNING  POSTPROCESSER :: Processing failed for /volume1/Downloads/sabnzbd/Complete/TV/Tokyo Ghoul Root A 01.mkv: 
2015-01-09 20:33:47 WARNING  POSTPROCESSER :: This show isn't in your list, you need to add it to SB before post-processing an episode
2015-01-09 20:33:47 DEBUG    POSTPROCESSER :: Unable to parse, skipping: Unable to parse TV Tokyo Ghoul Root A 01
2015-01-09 20:33:46 DEBUG    POSTPROCESSER :: Trying to find tv tokyo ghoul root a on TVRage
2015-01-09 20:33:45 DEBUG    POSTPROCESSER :: Trying to find tv tokyo ghoul root a on theTVDB
2015-01-09 20:33:45 DEBUG    POSTPROCESSER :: Using ALL regexes
2015-01-09 20:33:45 INFO     POSTPROCESSER :: Analyzing name u'TV Tokyo Ghoul Root A 01.mkv'
2015-01-09 20:33:45 DEBUG    POSTPROCESSER :: Unable to parse, skipping: Unable to parse /volume1/Downloads/sabnzbd/Complete/TV/Tokyo Ghoul Root A 01
2015-01-09 20:33:45 DEBUG    POSTPROCESSER :: Trying to find tokyo ghoul root a on TVRage
2015-01-09 20:33:45 DEBUG    POSTPROCESSER :: Trying to find tokyo ghoul root a on theTVDB
2015-01-09 20:33:45 DEBUG    POSTPROCESSER :: Using ALL regexes
2015-01-09 20:33:45 INFO     POSTPROCESSER :: Analyzing name u'/volume1/Downloads/sabnzbd/Complete/TV/Tokyo Ghoul Root A 01.mkv'
2015-01-09 20:33:45 DEBUG    POSTPROCESSER :: Unable to parse, skipping: Unable to parse TV
2015-01-09 20:33:45 DEBUG    POSTPROCESSER :: Using ALL regexes
2015-01-09 20:33:45 INFO     POSTPROCESSER :: Analyzing name u'TV'
2015-01-09 20:33:45 DEBUG    POSTPROCESSER :: Unable to parse, skipping: Unable to parse Tokyo Ghoul Root A 01
2015-01-09 20:33:45 DEBUG    POSTPROCESSER :: Trying to find tokyo ghoul root a on TVRage
2015-01-09 20:33:44 DEBUG    POSTPROCESSER :: Trying to find tokyo ghoul root a on theTVDB
2015-01-09 20:33:44 DEBUG    POSTPROCESSER :: Using ALL regexes
2015-01-09 20:33:44 INFO     POSTPROCESSER :: Analyzing name u'Tokyo Ghoul Root A 01.mkv'
2015-01-09 20:33:44 INFO     POSTPROCESSER :: Analyzing name None
2015-01-09 20:33:44 INFO     POSTPROCESSER :: Processing /volume1/Downloads/sabnzbd/Complete/TV/Tokyo Ghoul Root A 01.mkv (None)
2015-01-09 20:33:44 DEBUG    POSTPROCESSER :: PostProcessing VideoInRar: []
2015-01-09 20:33:44 DEBUG    POSTPROCESSER :: PostProcessing RarContent: []
2015-01-09 20:33:44 DEBUG    POSTPROCESSER :: PostProcessing VideoFiles: ['Tokyo Ghoul Root A 01.mkv']
2015-01-09 20:33:44 DEBUG    POSTPROCESSER :: PostProcessing Files: ['Tokyo Ghoul Root A 01.mkv']
2015-01-09 20:33:44 DEBUG    POSTPROCESSER :: PostProcessing Dirs: ['@eaDir']
2015-01-09 20:33:44 DEBUG    POSTPROCESSER :: PostProcessing Path: /volume1/Downloads/sabnzbd/Complete/TV
2015-01-09 20:33:44 DEBUG    POSTPROCESSER :: TV_DOWNLOAD_DIR: /volume1/Downloads/sabnzbd/Complete/TV
2015-01-09 20:33:44 DEBUG    POSTPROCESSER :: Processing folder /volume1/Downloads/sabnzbd/Complete/TV
2015-01-09 20:33:44 DEBUG    POSTPROCESSER :: Starting new thread: POSTPROCESSER
```

SR Version:	BRANCH: (master) / COMMIT: (33e95870e0c1b241c93f541d4ee27ed6ba3bfcb8) 

On Synology

**Comment by fernandog** [comment]
@yongbi85 still an issue?

**Comment by yongbi85** [comment]
Thanks it is ok now

Le lundi 16 février 2015, Fernando <notifications@github.com> a écrit :

> Closed #368 <https://github.com/SiCKRAGETV/sickrage-issues/issues/368>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SiCKRAGETV/sickrage-issues/issues/368#event-236084478>
> .
>


-- 
Envoyé depuis mon téléphone portable
